authorization/client_test.go

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:build !integration
```

Comment:
I think it is fine, if `make integration` will run Integration + Unit tests. I would remove this negation from all places.

Reply:
it means that the unit tests will be run twice but if it is ok then I will remove the negations

Reply:
Yes, unit tests are super fast and if we will want to count coverage on integration tests, it might be better.

If it will cause issues, we can add it back later. But for now I would skip that negation